topdown/http.go

```diff
@@ -44,6 +44,7 @@ var allowedKeyNames = [...]string{
 	"tls_client_key_env_variable",
 	"tls_client_cert_file",
 	"tls_client_key_file",
+	"skip_ssl",
```

Comment:
nit: Can we call the option `tls_insecure_skip_verify`, or `tls_allow_insecure` or something. Gist being to keep it familiar with the other `tls_*` options and similar to the OPA config option `allow_insecure_tls`